runtime/frequency/Cargo.toml

```diff
@@ -12,7 +12,7 @@ version = "0.0.0"
 targets = ["x86_64-unknown-linux-gnu"]
 
 [build-dependencies]
-substrate-wasm-builder = { version = "5.0.0-dev", git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.3.0"}
+substrate-wasm-builder = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.8.0"}
```

Comment:
Does it auto-set the version? why are we removing?

Reply:
If you’re using a specific branch from a Git repository, you can omit the version. However, including it doesn’t cause any issues. I’ll be creating a separate PR to switch to using crates instead of Git branches.